web.rb

```diff
@@ -341,11 +341,17 @@ def generate_payment_response(payment_intent)
   "👚" => 2500,
 }
 
+def price_lookup(product)
+  price = EMOJI_STORE[product]
+  raise "Can't find price for %s (%s)" % [product, product.ord.to_s(16)] if price.nil?
```

Comment:
should we have a default price for something instead of raising an exception?

Reply:
I’d like to know why it’s happening instead of silently ignoring it: The Emoji store items should line up between the server and the client, so breakage could indicate a problem with Unicode parsing or something.